osbrain/agent.py

```diff
@@ -193,20 +193,20 @@ class Agent():
         Default agent serialization format.
     transport : str, AgentAddressTransport, default is None
         Default agent transport protocol.
```

Comment:
You think name, UUID, host, serializer and transport should not be hidden?

Reply:
After hard deliberation I decided to only leave agent as public and make the others hidden.